tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -431,6 +443,7 @@ private void SetupAndRunTest(BsonDocument shared, BsonDocument test)
                     eventCapturer = new EventCapturer().Capture<CommandStartedEvent>(e => !DefaultCommandsToNotCapture.Contains(e.CommandName));
                 }
 
+                _objectMap = new Dictionary<string, object>();
```

Comment:
Currently, we have created objectMap collection in every test call

Reply:
Change not needed.

Reply:
done